TVOAppliance/hf_upgrade.sh

```diff
@@ -53,6 +53,13 @@ function check_package_status()
                 fi
         done
 }
+#function to change system settings
+function change_system_settings()
+{
+	grep -i dns /etc/ssh/sshd_config
+	sed -i '/UseDNS/c UseDNS no' /etc/ssh/sshd_config
+	grep -i dns /etc/ssh/sshd_config
```

Comment:
we don't need the grep commands as we don't need to show its output, sed command is good enough 
Also, add a single line comment above it mentioning what are we doing here

Reply:
Said changes are done.